testing/evaluate_tests.py

```diff
@@ -281,12 +281,15 @@ def evaluate_entities_and_schema(
     if verbose:
         logger.setLevel(logging.INFO)
 
-    entities = most_likely_entities = resp["intents"][num_intents]["entities"]
-    observed_entity_dict = {
-        # keep only the most likely hypothesis from Discovery -> first dict in list of dicts returned by Discovery
-        ent["label"]: ent["matches"][0]["value"]
-        for ent in entities  #most_likely_entities
-    }
+    if resp["intents"]:
+        entities = most_likely_entities = resp["intents"][num_intents]["entities"]
```

Comment:
@tshastry @mgoldey 

`entities = most_likely_entities = resp["intents"][num_intents]["entities"] if resp["intents"] else []`
to
`entities  = resp["intents"][num_intents]["entities"] if resp["intents"] else []`

do we want to keep the commented variable `most_likely_entities` ?

Reply:
I vote no